src/SimpleAzure.Storage.HybridQueue/HybridQueue.cs

```diff
@@ -70,7 +57,7 @@ public async Task AddMessageAsync<T>(T item, TimeSpan? initialVisibilityDelay, C
 
             var blobId = Guid.NewGuid().ToString(); // Unique Name/Identifier of this blob item.
             var binaryData = new BinaryData(message);
-            await _blobContainerClient.UploadBlobAsync(blobId, binaryData, cancellationToken);
+            await _blobContainerClient.UploadBlobAsync(blobId, binaryData, cancellationToken).ConfigureAwait(false);

```

Comment:
Since the project targets net7 and not net standard then this isn't needed as it wont be consumed in a .net framework project.

Reply:
I've never heard this advice, and I think maybe you have your wires crossed. Or maybe I do, because at the end of the day, basically all of my "real" work runs on .NET Framework.

Using configure await true means the continuation (the code after the await) will be queued back to the original context. This is true regardless of target framework. What changed in ASP.NET Core in recent versions is they don't have a custom SynchronizationContext so there is less chance of a deadlock when doing this queuing. That removes one of the reasons for configure await false, but there is still a small perf gain from using it, and it's still the default guidance (AFAIK) for library code, which this is.

If this code is called from a WinForms or WPF app, and doesn't use configure await false, then it's going to potentially deadlock, or just have to wait, to get back to the UI thread in order to deal with the results of uploading a blob to Azure, a task which objectively doesn't need the UI thread. Why take that hit?

Reply:
Seems I (and many others) were wrong about `CA(f)` with .NET "Core" .. thinking it applied to -all- scenario's .. not just the ASP part of ASP .NET "Core".

Approved.

I'm also crying and vomiting. Fucking hate seeing this "verbose ceremony" in my code base. Why not just have a fricking default setting that is "all await'd methods auto `.CF(f)` unless manually opting out .. or something.